build/includes/sdk.mk

```diff
@@ -77,7 +78,7 @@ run-sdk-command:
 		cd - ; \
 		$(MAKE) ensure-build-sdk-image SDK_FOLDER=$(SDK_FOLDER) ; \
 		docker run --rm $(common_mounts) -e "VERSION=$(VERSION)" \
-			$(DOCKER_RUN_ARGS) $(build_sdk_prefix)$(SDK_FOLDER):$(build_version) $(COMMAND) ; \
+			$(DOCKER_RUN_ARGS) $(DOCKER_PARAMETERS) $(build_sdk_prefix)$(SDK_FOLDER):$(build_version) $(COMMAND) ; \
```

Comment:
That is really strange I have added a clean step it would run as last `make run-sdk-conformance-tests` step.
I was checking that all is right, with `docker rmi ID`